app/renderer/js/components/tab.ts

```diff
@@ -27,17 +23,14 @@ class Tab extends BaseComponent {
 
 	activate(): void {
 		this.$el.classList.add('active');
-		this.webview.load();
 	}
 
 	deactivate(): void {
 		this.$el.classList.remove('active');
-		this.webview.hide();
 	}
 
 	destroy(): void {
```

Comment:
Are we using these listeners anywhere?
> activate
> deactivate
> destroy

If yes, please document this.

Reply:
Done.